tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
```

Comment:
**question**: could we make this a bit more recent? :P

Reply:
Fuel to the fire :P
https://www.freecodecamp.org/news/javascript-new-features-es2020/

Reply:
For sure!

But I think it's better to do this update at another time, as there would be a lot to validate in this PR

Reply:
@matheushdsbr Pls don't forget to open a new PR with this :)